webauthn4j-metadata/src/main/java/com/webauthn4j/metadata/data/statement/AuthenticationAlgorithm.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.webauthn4j.metadata.data.statement;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.exc.InvalidFormatException;
+import com.webauthn4j.util.UnsignedNumberUtil;
+
+/**
+ * The supported authentication algorithm(s).
+ * See section 3.6.1 Authentication Algorithms of FIDO Registry of Predefined Values
+ */
+public enum AuthenticationAlgorithm {
+
+    ALG_SIGN_SECP256R1_ECDSA_SHA256_RAW(0x0001),
+    ALG_SIGN_SECP256R1_ECDSA_SHA256_DER(0x0002),
+    ALG_SIGN_RSASSA_PSS_SHA256_RAW(0x0003),
+    ALG_SIGN_RSASSA_PSS_SHA256_DER(0x0004),
+    ALG_SIGN_SECP256K1_ECDSA_SHA256_RAW(0x0005),
+    ALG_SIGN_SECP256K1_ECDSA_SHA256_DER(0x0006),
+    ALG_SIGN_SM2_SM3_RAW(0x0007),
+    ALG_SIGN_RSA_EMSA_PKCS1_SHA256_RAW(0x0008),
+    ALG_SIGN_RSA_EMSA_PKCS1_SHA256_DER(0x0009),
+    ALG_SIGN_RSASSA_PSS_SHA384_RAW(0x000A),
+    ALG_SIGN_RSASSA_PSS_SHA512_RAW(0x000B),
+    ALG_SIGN_RSASSA_PKCSV15_SHA256_RAW(0x000C),
+    ALG_SIGN_RSASSA_PKCSV15_SHA384_RAW(0x000D),
+    ALG_SIGN_RSASSA_PKCSV15_SHA512_RAW(0x000E),
+    ALG_SIGN_RSASSA_PKCSV15_SHA1_RAW(0x000F),
+    ALG_SIGN_SECP384R1_ECDSA_SHA384_RAW(0x0010),
+    ALG_SIGN_SECP521R1_ECDSA_SHA512_RAW(0x0011),
+    ALG_SIGN_ED25519_EDDSA_SHA256_RAW(0x0012);
```

Comment:
Since it is not introduced as constants but as enum, "ALG_SIGN_" prefix is unnecessary for enum members.

Reply:
Should I also remove USER_VERIFY_ prefix?
https://github.com/webauthn4j/webauthn4j/issues/42

FIDO Registry of Predefined Values tells that "The USER_VERIFY constants are flags in a bitfield represented as a 32 bit long integer. ", therefore, USER_VERIFY is also constants same as ALG_SIGN.

Reply:
Yes, I overlooked it by mistake.

Reply:
Thank your for your comments.
BTW, should I remove ATTESTATION_ prefix from existing AttestationType.java ?
It seems that ATTESTATION is also constant mentioned in https://fidoalliance.org/specs/fido-v2.0-rd-20180702/fido-registry-v2.0-rd-20180702.html#authenticator-attestation-types

Reply:
Good catch. It needs to be addressed too.

Reply:
Hope I'll deal with it.